static/sass/_pattern_blog-topic.scss

```diff
@@ -0,0 +1,12 @@
+@mixin blog-p-topic {
+  .p-blog-topic__strip {
+    background-color: $color-accent-background;
```

Comment:
I don't understand why we are setting this colour and then inline in the mark-up

Reply:
you are right, sorry. Initially I tried to set the colour here as a variable but I realised that for u.c the accent colour is different than blog and I had to apply the colour inline... and forgot to remove this. Will amend it.